tripy/tripy/utils/stack_info.py

```diff
@@ -119,14 +119,17 @@ def get_stack_info(include_code_index: int = None) -> StackInfo:
             source_info._dispatch_target = frame.f_locals["key"]
 
         try:
-            # In Python 3.11, frames contain column offset information.
-            frame.f_code.co_positions
-        except AttributeError:
+            if not hasattr(frame.f_code, "co_positions"):
+                raise AttributeError
```

Comment:
I think we can just use `getattr` or `frame.f_code.co_positions` here. It's effectively the same as the condition.